dev-docs/modules/userId.md

```diff
@@ -1366,6 +1413,7 @@ Bidders that want to support the User ID module in Prebid.js, need to update the
 <div class="table-responsive" markdown="1">
 | ID System Name | ID System Host | Prebid.js Attr | Example Value |
 | --- | --- | --- | --- | --- | --- |
+| Admixer ID | Admixer | bidRequest.userId.__adm__admixer | `"1111"` |
```

Comment:
pending update based on my comments in the pbjs pr, this may need to change to `bidRequest.userId.admixerId` to follow conventions